src/test/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/controller/ImportControllerTest.kt

```diff
@@ -50,12 +51,26 @@ class ImportControllerTest(@Autowired val restTemplate: TestRestTemplate) {
         assertThat(locationRepository.count()).isEqualTo(0)
         assertThat(priceRepository.count()).isEqualTo(0)
 
-        val response = restTemplate.getForEntity("/generate-prices-spreadsheet/SERCO?moves_from=2020-10-01&moves_to=2020-10-01&reports_to=2020-10-01", InputStreamResource::class.java)
+        val response = restTemplate.getForEntity("/generate-prices-spreadsheet/SERCO?moves_from=2020-10-01&moves_to=2020-12-01&reports_to=2020-10-01", InputStreamResource::class.java)
 
         verify(spreadsheetProtection).protectAndGet(any())
         assertThat(response.headers.contentDisposition.filename).isEqualTo("Journey_Variable_Payment_Output_SERCO_2020-10-13_15_25.xlsx")
         assertThat(response.statusCode).isEqualTo(HttpStatus.OK)
         assertThat(locationRepository.count()).isEqualTo(2)
         assertThat(priceRepository.count()).isEqualTo(1)
     }
+
+    @Test
+    fun `fails if more than one months data is requested`() {
```

Comment:
Name of test is `fails if more than one months data is requested` but it's for more than 2 months

Reply:
Noted.  Will fix in future PR.